package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "genesys-cloud-streaming-client",
-  "version": "12.0.0",
+  "version": "12.0.1",
   "description": "client for the Genesys Cloud Streaming APIs (websocket/xmpp interface)",
   "repository": "https:github.com/purecloudlabs/genesys-cloud-streaming-client",
   "license": "MIT",
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
-  "main": "src/client.js",
+  "main": "dist/streaming-client.js",
```

Comment:
Had to change this to point at the node compatible, webpack built source file. Pointing straight at `src/client.js` caused a lot of headaches for build tools trying to load it in a node way (using commonjs).